src/streamlit_extras/card/__init__.py

```diff
@@ -0,0 +1,20 @@
+from streamlit_card import card
+from .. import extra
+
+card = extra(card)
+
+def example():
+    card(
+        title="Hello World!",
+        text="Some description",
+        image="http://placekitten.com/300/250"
+    )
+
+__title__ = "Streamlit Card"  # title of your extra!

```

Comment:
```suggestion
__title__ = "Card"  # title of your extra!
```

(otherwise we get "Streamlit" written likely everywhere)